src/precice_to_aste.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+import subprocess
+import argparse, logging
+
+def run(cmd):
+    print("+ " + cmd)
+    subprocess.run(cmd, shell = True, check = True)
```

Comment:
If you go the extra mile of wrapping the run command, then you could also capture stdout/err and output it only in case of a [`SubProccessError`](https://docs.python.org/3.7/library/subprocess.html#subprocess.SubprocessError).
This keeps everything nice and clean.